detections/endpoint/spoolsv_suspicious_loaded_modules.yml

```diff
@@ -0,0 +1,61 @@
+name: Spoolsv Suspicious Loaded Modules
+id: a5e451f8-da81-11eb-b245-acde48001122
+version: 1
+date: '2021-07-01'
+author: Splunk Threat Research Team
+type: batch
+datamodel:
+- Endpoint
+description: This search is to detect suspicious loading of dll in specific path relative to printnightmare exploitation.
+  In this search we try to detect the loaded modules made by spoolsv.exe after the exploitation. 
+search: '`sysmon` EventCode=7 Image ="*\\spoolsv.exe" ImageLoaded="*\\Windows\\System32\\spool\\drivers\\x64\\*" ImageLoaded = "*.dll"  
+  | stats dc(ImageLoaded) as countImgloaded values(ImageLoaded) as ImgLoaded count min(_time) as firstTime max(_time) as lastTime by Image Computer EventCode 
```

Comment:
should we grab the process id out of this, with a PID something downstream to use like phantom.